x-pack/solutions/observability/plugins/observability/public/pages/overview/components/sections/ux/ux_section.tsx

```diff
@@ -79,9 +77,10 @@ export function UXSection({ bucketSize }: Props) {
     ]
   );
 
-  if (!uxHasDataResponse?.hasData) {
+  if (!uxHasDataResponse?.hasData || !services.exploratoryView) {
```

Comment:
This will also prevent the `CoreVitals` component from being rendered, is that the expected behaviour?